src/lib.rs

```diff
@@ -0,0 +1,336 @@
+extern crate wapc_guest as guest;
+use guest::prelude::*;
+
+use anyhow::anyhow;
+use k8s_openapi::api::core::v1 as apicore;
+use serde::{Deserialize, Serialize};
+use serde_json::Result;
+
+mod settings;
+use settings::Settings;
+
+mod taint;
+use taint::Taint;
+
+#[derive(Deserialize, Serialize, Debug)]
```

Comment:
```suggestion
#[derive(Serialize, Debug)]
```

Reply:
The `Deserialize` trait is needed to run the unit tests